artview/components/nav.py

```diff
@@ -133,19 +135,19 @@ def createNavToolbar(self):
         pixlast = QtGui.QPixmap(os.sep.join([parentdir, 'icons',"arrow_go_last_icon.png"]))
         self.act_first = self.navtoolbar.addAction(
             QtGui.QIcon(pixfirst),
-            "First file: %s"%(os.path.basename(self.Vfilelist.value[0])),
+            "First file:",
             self.goto_first_file)
         self.act_prev = self.navtoolbar.addAction(
             QtGui.QIcon(pixprev),
-            "Previous file: %s"%(os.path.basename(self.Vfilelist.value[self.fileindex - 1])),
+            "Previous file:",
             self.goto_prev_file)
         self.act_next = self.navtoolbar.addAction(
             QtGui.QIcon(pixnext),
-            "Next file: %s"%(os.path.basename(self.Vfilelist.value[self.fileindex + 1])),
+            "Next file:",
             self.goto_next_file)
         self.act_last = self.navtoolbar.addAction(
             QtGui.QIcon(pixlast),
-            "Last file: %s"%(os.path.basename(self.Vfilelist.value[-1])),
+            "Last file:",
             self.goto_last_file)
 
```

Comment:
Okay, so you fall back to this string upon testing, e.g. first file in a directory.